base/div.jl

```diff
@@ -80,6 +80,16 @@ See also: [`div`](@ref)
 julia> fld(7.3,5.5)
 1.0
 ```
+Because of floating-point rounding `fld(x,y)` can lead to an *alleged* violation of the `fld(x,y) == x/y` condition:
```

Comment:
```suggestion
Because `fld(x, y)` implements strictly correct floored rounding based on the true of floating-point numbers, unintuitive situations can arise. For example:
```